README/ReleaseNotes/v620/index.md

```diff
@@ -364,6 +364,10 @@ prevent infinite loops or crashes.
   release.
 - The version of Python that ROOT has been built with is now exported in the
   CMake variable `${ROOT_PYTHON_VERSION}`, available after calling `find_package(ROOT)`.
+- python-dev is now required and the compilation will fail if CMake is not able
```

Comment:
Please place this bullet as the second in this section (right after the one that says that the `python` option is deprecated).